addons/website_sale_delivery/views/delivery_carrier_views.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record model="ir.ui.view" id="view_delivery_carrier_form">
-        <field name="name">delivery.carrier.website.form</field>
+        <field name="name">delivery.carrier.form.inherit</field>
```

Comment:
This change of name makes it less clear what this view is about. It may be useful to say it is an inherit but don't remove the module.